avocado/utils/network/interfaces.py

```diff
@@ -313,6 +318,36 @@ def save(self, ipaddr, netmask):
                       'IPV6_DEFROUTE': 'yes'}
         if current_distro.name == 'SuSE':
             ifcfg_dict.pop('BOOTPROTO')
+
+        if self.if_type == 'Bond':
+            # Update master and slave cfg for bond
+            ifcfg_dict.update({'BONDING_MASTER': 'yes'})
+            mode = self._get_bondinterface_details().splitlines()[0].split()[0]
+            slaves = self._get_bondinterface_details().splitlines()[1].split()
+            ifcfg_slave_dict = {'SLAVE': 'yes',
+                                'ONBOOT': 'yes',
+                                'MASTER': self.name}
+            if current_distro.name == 'SuSE':
+                ifcfg_dict.update({'BONDING_MODULE_OPTS=mode': mode})
```

Comment:
This looks weird to me... don't you mean:

```
ifcfg_dict.update({'BONDING_MODULE_OPTS': mode})
```

Or, according to the previous suggestion:

```python
ifcfg_dict['BONDING_MODULE_OPTS'] = mode
```